docs/_scripts/feature-selection.py

```diff
@@ -0,0 +1,138 @@
+from pathlib import Path
+
+_file = Path(__file__)
+print(f"Executing {_file}")
+
+_static_path = Path("_static") / _file.stem
+_static_path.mkdir(parents=True, exist_ok=True)
+
+import matplotlib.pyplot as plt
```

Comment:
I know this is WIP, but can we use seaborn style as in the other plots?

```suggestion
import matplotlib.pyplot as plt
import seaborn as sns

sns.set_theme()
```